kitsune/sumo/static/sumo/js/ui.js

```diff
@@ -275,6 +276,14 @@
     }
   }
 
+  function correctFixedHeader(){
+    var headerHeight = document.getElementsByTagName("header");
```

Comment:
This should be `var headerHeight = document.querySelector('header');`. Using `querySelector` universally will make things a bit easier to read.